### ISSUE TITLE
Could not find file

### ISSUE THREAD

**Issue by colinf** [trevordevore/sublimelinter-contrib-livecodelint#1]
It could well just be a mistake that I have made during install but with ST3 on Mac I am getting the following error:

SublimeLinter: livecodelint output:
  row 0, col 0: include: could not find file (/Users/cfallon/Library/Application Support/Sublime Text 3/Installed Packages/SublimeLinter-contrib-livecodelint.sublime-package/livecodelint.lc)

SublimeLinter and this package have been installed and the package is in the directory in the error message. But for some reason it doesn't seem to work.

**Comment by colinf** [comment]
For info, it works if I follow the manual installation instructions rather than using the Sublime package control install

**Comment by trevordevore** [comment]
@colinf Did you restart ST3 during the manual installation process? I wonder if perhaps a restart is required in order for the listing to start working?

**Comment by colinf** [comment]
Hi Trevor, I did try restarting during the package control install. I've noticed that when I install with package control it installs a sublime package file into the packages directory rather than a folder. So there is a file .../Installed Packages/SublimeLinter-contrib-livecodelint.sublime-package but this is a package file so the there is no file .../Installed Packages/SublimeLinter-contrib-livecodelint.sublime-package/livecodelint.lc at the OS level as livecodelint.lc is actually within the ST package file.
